tests/binary-expressions/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -8,12 +8,11 @@ function f() {
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 function f() {
-  const appEntitys = getAppEntitys(loadObject).filter(
-    entity =>
-      entity &&
-      entity.isInstallAvailable() &&
-      !entity.isQueue() &&
-      entity.isDisabled()
+  const appEntitys = getAppEntitys(loadObject).filter(entity =>
```

Comment:
I personally prefer the old behaviour.

I tried briefly to only apply the group `ifBreak`, but I'm not sure that's possible (`willBreak` is insufficient here).